nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -602,54 +598,34 @@ define([
             const meta = cell.metadata;
             meta.kbase.attributes.lastLoaded = new Date().toUTCString();
             cell.metadata = meta;
-            render()
-                .then(() => {
-                    // add in the control panel so we can update the job status in the cell header
-                    jobManager.addHandler('modelUpdate', {
-                        controlPanel: (jobManagerContext) => {
-                            // Update the execMessage panel with details of the active jobs
-                            controlPanel.setExecMessage(
-                                Jobs.createCombinedJobState(
-                                    jobManagerContext.model.getItem('exec.jobs.byStatus')
-                                )
-                            );
-                        },
-                        fsmState: (jobManagerContext) => {
-                            const fsmState = Jobs.getFsmStateFromJobs(
+            render().then(() => {
+                // add in the control panel so we can update the job status in the cell header
+                jobManager.addHandler('modelUpdate', {
+                    controlPanel: (jobManagerContext) => {
+                        // Update the execMessage panel with details of the active jobs
+                        controlPanel.setExecMessage(
+                            Jobs.createCombinedJobState(
                                 jobManagerContext.model.getItem('exec.jobs.byStatus')
-                            );
-                            if (fsmState) {
-                                updateState(fsmState);
-                            }
-                        },
-                    });
-
-                    // TODO: assess cell state, update job info if required
-                    // jobManager.restorefromSaved()
+                            )
+                        );
+                    },
+                    fsmState: (jobManagerContext) => {
+                        const fsmState = Jobs.getFsmStateFromJobs(
+                            jobManagerContext.model.getItem('exec.jobs.byStatus')
+                        );
+                        if (fsmState) {
+                            updateState(fsmState);
+                        }
+                    },
+                });
 
-                    return BulkImportUtil.evaluateConfigReadyState(model, specs);
-                })
-                .then((appReadyState) => {
-                    const curState = model.getItem('state');
-                    const curReadyState = curState.params;
-                    const updatedReadyState = !SimpleUtil.isEqual(appReadyState, curReadyState);
```

Comment:
so most of this is now no longer necessary?

Reply:
Correct, at least I think so. 
That block runs on cell startup (and is only really relevant after a page reload) and does a check over the app inputs to see what state they're in. This is fine, except for two things:

1. Loading the configure tab would do it again.
2. It's completely irrelevant if the cell is being run, or has been run.

So having this here is just redundant at this point. The other problem is that it would require a good chunk of rearranging where information goes and where it comes from so that this could get run. The configure tab has all of the information already at hand, and it would take some data munging (and running a `list` command against the staging service) to make it work here.

If it becomes a problem, I think we should move that check out entirely to a utility function that would just take in the cell's data model and figure it out, but I would worry about efficiency.